qa/rpc-tests/mempool_tx_expiry.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python2
```

Comment:
iiuc there is no test that checks that a block with expired transaction would be rejected by a fellow miner.
Should we add such a test?